README.md

```diff
@@ -25,41 +25,19 @@ advantage of performant hardware to speed up their simulation and exploration.
 
 Check the following pages for additional details on how to use them with
 **Inductiva API**:
-- [SPlisHSPlasH](https://github.com/inductiva/inductiva/wiki/SPlisHSPlasH)
-- [DualSPHysics](https://github.com/inductiva/inductiva/wiki/DualSPHysics)
-- [OpenFOAM](https://github.com/inductiva/inductiva/wiki/OpenFOAM)
-- [SWASH](https://github.com/inductiva/inductiva/wiki/SWASH)
-- [XBeach](https://github.com/inductiva/inductiva/wiki/XBeach)
-- [Reef3D](https://github.com/inductiva/inductiva/wiki/Reef3D)
-- [GROMACS](https://github.com/inductiva/inductiva/wiki/GROMACS)
-- [FDS](https://github.com/inductiva/inductiva/wiki/FDS)
+- [SPlisHSPlasH](https://github.com/inductiva/inductiva/blob/main/docs/simulators/SPlisHSPlasH.md)
```

Comment:
will this integrate well with readthedocs? we might need some relative paths or something, no?